### PR TITLE
On-ramp: Add what's new modal content

### PR DESCRIPTION
**Pull request by wachunei**
**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://docs.google.com/document/d/1VJLwTRsUw_5EDq_o8d6sSbXUAYENLurkRitYO45eY5o/edit?usp=sharing)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add `needs-qa` label when dev review is completed
5. Add `QA Passed` label when QA has signed off

**Description**

This PR
- makes pages indicator dynamic, functional (touchable) and visible only when slides > 1
- adds the content for the new on-ramp features, [more context in this slack thread](https://consensys.slack.com/archives/CBW7S9FSN/p1678825072396379)

**Screenshots/Recordings**

https://user-images.githubusercontent.com/1024246/226704184-0a4f4c5c-8fc3-4617-a4a8-d87529da637a.mov

**Issue**

Progresses #???

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

File: app/components/UI/WhatsNewModal/whatsNewList.ts [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+import Routes from '../../../constants/navigation/Routes';
 import { WhatsNew } from './types';
 
 export const whatsNew: WhatsNew = {
   // All users that have <5.4.0 and are updating to >=5.4.0 should see
-  onlyUpdates: true, // Only users who updated the app will see this, not newly installs
-  maxLastAppVersion: '5.4.0', // Only users who had a previous version <5.4.0 version will see this
-  minAppVersion: '5.4.0', // Only users who updated to a version >= 5.4.0 will see this
+  onlyUpdates: false, // Only users who updated the app will see this, not newly installs
+  maxLastAppVersion: '6.4.0', // Only users who had a previous version <5.4.0 version will see this
```
Comment (Cal-L) [inline app/components/UI/WhatsNewModal/whatsNewList.ts:9]: Can we update the comments to either match the versions being used or change to remove version dependencies

File: locales/languages/en.json [hunk 2]
```diff
@@ -2060,14 +2060,19 @@
...
-    "feature_token_security_button_text": "Let's start!"
+    "slides_button": "Buy crypto",
+    "slide_1_subtitle": "Forget centralized exchanges. Buying crypto with MetaMask just got easier.",
+    "slide_1_body_1": "Why not fund your MetaMask wallet directly? We’ve been busy onboarding new partners and local payment methods to make your fiat-to-crypto purchases with MetaMask much easier.",
+    "slide_1_body_2": "With 189 supported countries across eight different networks including L2's, we’ve got you covered. Select your preferred payment option and choose the best available quote.",
+    "slide_1_body_3": "Keep scrolling → to see details by region.",
+    "slide_2_subtitle": "Hello India!",
+    "slide_2_body_1": "Users in India can now access UPI & IMPS for cheaper, faster, and more efficient payments. Together with Transak and Onramp.money, we are making web3 more accessible than ever in India.",
+    "slide_3_subtitle": "Enabling PayID, POLi, Interac, and iDEAL!",
+    "slide_3_body_1": "Weve integrated with Banxa to offer local banking infrastructure and these new payment methods in Canada, Netherlands, Australia, South Africa, and Turkey. Check them out!",
```
Comment (Cal-L) [inline locales/languages/en.json:2071]: We've

**Review by Cal-L — APPROVED** [review]
Other than a couple comments - LGTM. Are we awaiting translated files?

File: locales/languages/en.json (hunk 2 above)
Comment (wachunei) [reply locales/languages/en.json:2071]: Thank you!

Comment (wachunei) [reply locales/languages/en.json]: fixed in [5618492](https://github.com/MetaMask/metamask-mobile/pull/6009/commits/56184924a2bbcaeca723792d1b291a2f99bb0900)

**Comment by wachunei** [comment]
Yes, we want these to be translated.

**Review by Cal-L — APPROVED** [review]
Translations LGTM

**Comment by chrisleewilcox** [comment]
@wachunei minor bug.  Breadcrumb does not highlight/select on 4th page.  Not able to highlight fifth page.

https://recordit.co/Fmebq1u4uV

Expected: breadcrumb selection follows all pages.

**Comment by wachunei** [comment]
I can't reproduce the bug 😩

Here is the feature working on iphone 11 and 14 Pro

https://user-images.githubusercontent.com/1024246/230246026-1f03ea75-900e-46be-8613-49fdb691b0d9.mp4


https://user-images.githubusercontent.com/1024246/230246030-a6dd6adb-3a49-47e4-b8a8-b4a0e98ef468.mp4

**Pull request merged**